source/features/index.tsx

```diff
@@ -104,12 +105,74 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		document.head.append(<style>{options.customCSS}</style>);
 	}
 
+	Object.assign(options, await bisectFeatures());
+
 	// Create logging function
 	log = options.logging ? console.log : () => {/* No logging */};
 
 	resolve(options);
 });
 
+async function bisectFeatures(): Promise<Record<string, boolean>> {
```

Comment:
This and related functions should be moved to their own file in `source/helpers` or something